about/updates/roadmap/index.md

```diff
@@ -17,54 +17,53 @@ sidenav: about_us
 
 At Search.gov we aim to provide an excellent public search experience, a self-service search administration experience, and a rock-solid secure and compliant infrastructure to support it all, while keeping costs down across the government.
 
-We know it’s helpful for our partners at federal agencies to know what’s coming up next, to support planning for your own web platforms. 
+We know it's helpful for our partners at federal agencies to know what’s coming up next, to support planning for your own web platforms. 
```

Comment:
@amanifarooque will this straight apostrophe display differently than the curly/angled apostrophe later in the sentence?

Reply:
Trying to make these consistent, made a couple more replacements.

Reply:
ok, thanks